docs/nx-cloud/recipes/access-tokens.md

```diff
@@ -1,31 +1,31 @@
-# Nx CLI and Access Tokens
+# Nx CLI and CI Access Tokens
 
-The permissions and membership define what developers can access on nx.app. They don't affect what happens when you run Nx commands locally or on CI. To manage that, you need to provision access tokens. To do that, go to Workspace Options / Manage Access Tokens.
+The permissions and membership define what developers can access on nx.app but they don't affect what happens when you run Nx commands in CI. To manage that, you need to provision CI access tokens in Workspace settings / Manage CI access tokens.
```

Comment:
1. I'm not a fan of this paragraph in general. I think we can be more clear in saying "CI Access Tokens controls permissions for Nx Commands in CI environments.